src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,148 @@
+package duke;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+import java.util.Arrays;
+
+public class Parser {
+    public String[] words; // user input split by spaces to get words
+
+    public Parser(String command, TaskList tasks, Storage storage) throws IOException {
```

Comment:
This method is a bit long - imagine if you have more commands to be added to the app, then it will be even longer. I would suggest that you can further abstract the implementation of each command into classes or methods.